web/client/epics/controls.js

```diff
@@ -16,5 +18,13 @@ module.exports = {
                 return Rx.Observable.of(action.action);
             }
             return Rx.Observable.of(action.elseAction.call());
-        })
+        }),
+
+    setControlPropertiesEpic: (action$) =>
+        action$
```

Comment:
I think this epic belongs to GFI because it's related to the behaviour of Identify. 
I think it's better to move this epic in identify epic and use a more specific name eg. `enabledCatalogClosesIdentifyEpic`
@offtherailz what do you think?

Reply:
Noted